apps/meteor/tests/end-to-end/api/18-oauthapps.js

```diff
@@ -161,4 +161,90 @@ describe('[OAuthApps]', function () {
 				});
 		});
 	});
+
+	describe('[/oauth-apps.update]', () => {
+		let appId;
+
+		before((done) => {
+			const name = 'test-oauth-app';
+			const redirectUri = 'https://test.com';
+			const active = true;
+			request
+				.post(api('oauth-apps.create'))
+				.set(credentials)
+				.send({
+					name,
+					redirectUri,
+					active,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.end((err, res) => {
+					appId = res.body.application._id;
+					done();
+				});
+		});
+
+		it('should update an app name, active and redirect URI correctly by its id', async () => {
+			const name = `new app ${Date.now()}`;
+			const redirectUri = 'http://localhost:3000';
+			const active = false;
+
+			await request
+				.post(api(`oauth-apps.update`))
+				.set(credentials)
+				.send({
+					appId,
+					name,
+					redirectUri,
+					active,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body).to.have.property('active', active);
+					expect(res.body).to.have.property('redirectUri', redirectUri);
+					expect(res.body).to.have.property('name', name);
+				});
+		});
+	});
+
+	describe('[/oauth-apps.delete]', () => {
+		let appId;
+
+		before((done) => {
+			const name = 'test-oauth-app';
+			const redirectUri = 'https://test.com';
+			const active = true;
+			request
+				.post(api('oauth-apps.create'))
+				.set(credentials)
+				.send({
+					name,
+					redirectUri,
+					active,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.end((err, res) => {
+					appId = res.body.application._id;
+					done();
+				});
+		});
+
+		it('should delete an app by its id', async () => {
+			await request
+				.delete(api(`oauth-apps.delete`))
```

Comment:
```suggestion
				.post(api(`oauth-apps.delete`))
```
If we decide to change the delete endpoint to POST, then this should be updated too.